### PR TITLE
Created Delete API

### PR DESCRIPTION
**Pull request by DrakeEsdon**
Screenshot from 2019-07-27 15-43-04
Screenshot from 2019-07-27 15-43-31
Screenshot from 2019-07-27 15-43-42

### CODE REVIEW

**Review by HugoKlepsch — APPROVED** [review]
LGTM :+1:

File: server/serverRun.py [hunk 1]
```diff
@@ -220,6 +220,25 @@ def items():
     return json.dumps(user_items), 200, JSON_CT
 
 
+@app.route('/item/<int:item_id>',methods=['DELETE'])
+def delete_item(item_id):
+    user_id = session['user_id']
+    try:
+        picRow = Picture.query.filter_by(id=item_id).one()
```
Comment (HugoKlepsch) [inline server/serverRun.py]: Just realized this may not work. There can be more than one picture per item.

**Review by HugoKlepsch — CHANGES_REQUESTED** [review]

File: server/serverRun.py (hunk 1 above)
Comment (HugoKlepsch) [reply server/serverRun.py]: Also, cascade delete would be nice.

**Review by DrakeEsdon — COMMENTED** [review]
Screenshot from 2019-07-27 16-32-40

File: server/models.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ class Item(db.Model):
     sell_price = db.Column(db.Numeric(precision=19, scale=4, asdecimal=True), nullable=True)
     listed_price = db.Column(db.Numeric(precision=19, scale=4, asdecimal=True), nullable=True)
 
-    pictures = db.relationship('Picture', backref='item', lazy=True)
+    children = db.relationship("Picture", cascade="all,delete", backref="item")
```
Comment (HugoKlepsch) [inline server/models.py]: I think this should still be called pictures

**Review by HugoKlepsch — APPROVED** [review]
Ok this is good now.

**Pull request merged**